packages/client/src/devices/InputMediaDeviceManager.ts

```diff
@@ -20,6 +20,10 @@ export abstract class InputMediaDeviceManager<
    * @internal
    */
   disablePromise?: Promise<void>;
+  /**
+   * if true, stops the media stream when call is left
+   */
+  stopOnLeave = false;
```

Comment:
hey @santhoshvai, Wouldn't it be better if we set this to `true` by default?

Reply:
well, yes, sorry was testing them and missed to change it